sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsRequestOptions.java

```diff
@@ -54,4 +55,24 @@ public TextAnalyticsRequestOptions setIncludeStatistics(boolean includeStatistic
         this.includeStatistics = includeStatistics;
         return this;
     }
+
+    /**
+     * Get the value of {@code domain}.
+     *
+     * @return the value of domain.
+     */
+    public String getDomain() {
+        return domain;
+    }
+
+    /**
+     * Set the value of {@code domain}.
+     *
+     * @param domain (Optional) if set to 'PHI', response will contain only PHI entities.
+     * @return the {@link TextAnalyticsRequestOptions} object itself.
+     */
+    public TextAnalyticsRequestOptions setDomain(String domain) {
```

Comment:
also, the way it is presented here, it seems like other endpoints could use the `domain` property too. I'm still unclear of the scope of this property so better leave it out until we fully understand it

Reply:
yup. Will leave it out for now.